### ISSUE TITLE
[v2 QUESTION]: Replicates vs Raytune Trials during Hyperparameter Optimization

### ISSUE THREAD

**Issue by spyda90** [chemprop/chemprop#1258]
Hi,

I am trying to deeply understand the options in `hpopt` in v2.
I have understand that the number of trials is controlled by `--raytune-num-samples`.
Some questions:

- Can I set a seed for configuration used in the different trials?
- I am using `--ensemble 5` but I am not sure ensemble is actually running. Is ensemble allowed in hpopt (as it was in v1)?
- I have tried to use `--num-replicates` but I am not sure different replicates are run for each trial. In case --num-replicates is not allowed/used, and in absence of CV, how can I be sure that the best performance (best configuration) can be recapitulated during training keeping the same splitting? I have tried to use `--save-smiles-splits ` but no CSV files have been written (using -`-split-type random`)

More in general, I am trying to understand how maximize hpopt performance (e.g. increasing number of trials) while keeping consistency going from hpopt to training.

Thank you in advance

**Comment by JacksonBurns** [comment]
Only the first replicate split is used, see: https://chemprop.readthedocs.io/en/latest/tutorial/cli/hpopt.html#splitting

I think ensemble is ignored, though, based on the fact that it is only present in [training CLI](https://github.com/chemprop/chemprop/blob/c4602da66bf9dc766ea48af126fc2600b445e923/chemprop/cli/train.py#L191).

You can check this by running `hpopt` with some tiny training set (like, 10 rows) and set both of these flags to some number. If the total number of trained models is the product of these numbers, it's using both, etc.

**Comment by spyda90** [comment]
Hi Jackson,

Thank you very much. I'll close this one.
